tests/src/python/test_qgssymbollayer.py

```diff
@@ -579,6 +635,12 @@ def testQgsSimpleFillSymbolLayer(self):
         mMessage = 'Expected "%s" got "%s"' % (mExpectedValue, mValue)
         assert mExpectedValue == mValue, mMessage
 
+        ctx = QgsRenderContext()
+        mExpectedValue = 0
+        mValue = len(mSymbolLayer.usedAttributes(ctx))
+        mMessage = 'Expected no attributes used'
+        assert mExpectedValue == mValue, mMessage
```

Comment:
is this not possible?
It normally gives better messages for debugging and is more modern.

```suggestion
        self.assertEqual(mExpectedValue, mValue)
```

Reply:
This code is conformed to all the tests in this file. Do you think we have to update all the tests ?

Reply:
That would be nice but I didn't dare to ask that from you.

Reply:
Ideally, but at least we can avoid introducing more of the old style test approach

Reply:
Ok, thanks for the review